lib/operations/mongo_client_ops.js

```diff
@@ -103,7 +103,15 @@ const validOptionNames = [
   'retryWrites',
   'useNewUrlParser',
   'useUnifiedTopology',
-  'optionsValidationLevel'
+  'optionsValidationLevel',
```

Comment:
Note that the `validOptions` function should go away once options management is fully integrated. `validOptions` is only responsible for warning on unknown and deprecated options. I added these options here to prevent test failures, but I will remove all of this after [NODE-1381](https://jira.mongodb.org/browse/NODE-1381) is resolved.

Reply:
:+1: